agent/config/config.go

```diff
@@ -113,6 +116,16 @@ type Plugin struct {
 	Config map[string]string `hcl:"config,optional"`
 }
 
+// Policy holds the configuration information specific to the policy manager
+// and resulting policy parsing.
+type Policy struct {
```

Comment:
maybe this is just me, but... I know that this is `config.Policy`, which is a distinct object from `Policy.Policy`. but it is a little confusing (at least, in GitHub PRs) that they have the same name. just my two cents. by the same token, `config.PolicyConfig` would be a little redundant.

Reply:
how does `PolicyManager` sound?

Reply:
i think I'm going to retract this comment.